brainstormforce/astra
# AST-939 - Offcanvas content directly visible on frontend when Toggle Button not dropped in builder area

### Description
- Offcanvas content appears on frontnend when Toggle Button not added in the header builder

### Screenshots
- https://share.bsf.io/KouJ80P6

### Types of changes





### How has this been tested?
- Check this issue also resolves when imported above customizer JSON - https://share.bsf.io/jku4jNG0
- Check this main issue (by non adding mobile toggle component in header builder)
- Check toggle for desktop added and by removing from mobile layouts
- Test different header offcanvas types

### Checklist:
